src/App.js

```diff
@@ -1,29 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+// eslint-disable-next-line import/extensions
+import Calculator from './components/Calculator.js';
```

Comment:
* Kindly fix this error I get when I try to start the app as I have highlighted in the screenshot below you are importing `Calculator` from `./components` in lowercase rather than uppercase as you have it in your file structure, please change `./components` to uppercase and this error will be solved 👍 
![image](https://user-images.githubusercontent.com/52497246/198200927-4454bb75-827d-492a-b2ed-f2442428ae09.png)